{
  "url": "https://api.github.com/repos/matter-labs/zksync/issues/244",
  "repository_url": "https://api.github.com/repos/matter-labs/zksync",
  "labels_url": "https://api.github.com/repos/matter-labs/zksync/issues/244/labels{/name}",
  "comments_url": "https://api.github.com/repos/matter-labs/zksync/issues/244/comments",
  "events_url": "https://api.github.com/repos/matter-labs/zksync/issues/244/events",
  "html_url": "https://github.com/matter-labs/zksync/issues/244",
  "id": 695063272,
  "node_id": "MDU6SXNzdWU2OTUwNjMyNzI=",
  "number": 244,
  "title": "database incompatibility (rinkeby branch)",
  "user": {
    "login": "scream4ik",
    "id": 1583040,
    "node_id": "MDQ6VXNlcjE1ODMwNDA=",
    "avatar_url": "https://avatars.githubusercontent.com/u/1583040?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/scream4ik",
    "html_url": "https://github.com/scream4ik",
    "followers_url": "https://api.github.com/users/scream4ik/followers",
    "following_url": "https://api.github.com/users/scream4ik/following{/other_user}",
    "gists_url": "https://api.github.com/users/scream4ik/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/scream4ik/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/scream4ik/subscriptions",
    "organizations_url": "https://api.github.com/users/scream4ik/orgs",
    "repos_url": "https://api.github.com/users/scream4ik/repos",
    "events_url": "https://api.github.com/users/scream4ik/events{/privacy}",
    "received_events_url": "https://api.github.com/users/scream4ik/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "closed",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 1,
  "created_at": "2020-09-07T12:43:22Z",
  "updated_at": "2020-09-11T11:55:28Z",
  "closed_at": "2020-09-11T11:55:28Z",
  "author_association": "NONE",
  "active_lock_reason": null,
  "body": "Maybe it's ok and you know about it. When I try to run zksync using rinkeby branch, I get an error:\r\n```\r\npostgres_1    | 2020-09-07 10:51:44.379 UTC [1] FATAL:  database files are incompatible with server\r\npostgres_1    | 2020-09-07 10:51:44.379 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 10, which is not compatible with this version 12.4 (Debian 12.4-1.pgdg100+1).\r\n```\r\nWith master branch all fine",
  "closed_by": {
    "login": "scream4ik",
    "id": 1583040,
    "node_id": "MDQ6VXNlcjE1ODMwNDA=",
    "avatar_url": "https://avatars.githubusercontent.com/u/1583040?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/scream4ik",
    "html_url": "https://github.com/scream4ik",
    "followers_url": "https://api.github.com/users/scream4ik/followers",
    "following_url": "https://api.github.com/users/scream4ik/following{/other_user}",
    "gists_url": "https://api.github.com/users/scream4ik/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/scream4ik/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/scream4ik/subscriptions",
    "organizations_url": "https://api.github.com/users/scream4ik/orgs",
    "repos_url": "https://api.github.com/users/scream4ik/repos",
    "events_url": "https://api.github.com/users/scream4ik/events{/privacy}",
    "received_events_url": "https://api.github.com/users/scream4ik/received_events",
    "type": "User",
    "site_admin": false
  },
  "reactions": {
    "url": "https://api.github.com/repos/matter-labs/zksync/issues/244/reactions",
    "total_count": 0,
    "+1": 0,
    "-1": 0,
    "laugh": 0,
    "hooray": 0,
    "confused": 0,
    "heart": 0,
    "rocket": 0,
    "eyes": 0
  },
  "timeline_url": "https://api.github.com/repos/matter-labs/zksync/issues/244/timeline",
  "performed_via_github_app": null,
  "state_reason": "completed"
}
[
  {
    "url": "https://api.github.com/repos/matter-labs/zksync/issues/comments/688678656",
    "html_url": "https://github.com/matter-labs/zksync/issues/244#issuecomment-688678656",
    "issue_url": "https://api.github.com/repos/matter-labs/zksync/issues/244",
    "id": 688678656,
    "node_id": "MDEyOklzc3VlQ29tbWVudDY4ODY3ODY1Ng==",
    "user": {
      "login": "dvush",
      "id": 21145096,
      "node_id": "MDQ6VXNlcjIxMTQ1MDk2",
      "avatar_url": "https://avatars.githubusercontent.com/u/21145096?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/dvush",
      "html_url": "https://github.com/dvush",
      "followers_url": "https://api.github.com/users/dvush/followers",
      "following_url": "https://api.github.com/users/dvush/following{/other_user}",
      "gists_url": "https://api.github.com/users/dvush/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/dvush/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/dvush/subscriptions",
      "organizations_url": "https://api.github.com/users/dvush/orgs",
      "repos_url": "https://api.github.com/users/dvush/repos",
      "events_url": "https://api.github.com/users/dvush/events{/privacy}",
      "received_events_url": "https://api.github.com/users/dvush/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2020-09-08T07:32:25Z",
    "updated_at": "2020-09-08T07:32:25Z",
    "author_association": "CONTRIBUTOR",
    "body": "Remove volumes directory and make zksync init again.",
    "reactions": {
      "url": "https://api.github.com/repos/matter-labs/zksync/issues/comments/688678656/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  }
]
